package.json

```diff
@@ -7,10 +7,11 @@
     "benchmark-server": "node tests/benchmark-server/index.js",
     "build": "node tasks/build.js --release",
     "build:all": "node tasks/build.js --debug --release --api",
-    "code-style": "eslint --ignore-pattern \"!.eslintplugin.js\" --cache --fix \"src/**/*.ts\" \"src/**/*.tsx\" \"tasks/**/*.js\" \"tests/[!coverage]**/*.js\" \"tests/**/*.ts\" \".eslintrc.js\" \"index.d.ts\" \".eslintplugin.js\"",
+    "code-style": "eslint --ignore-pattern '!.eslintplugin.js' --cache --fix -- 'src/**/*.ts' 'src/**/*.tsx' 'tasks/**/*.js' 'tests/[!coverage]**/*.js' 'tests/**/*.ts' '.eslintrc.js' 'index.d.ts' '.eslintplugin.js'",
```

Comment:
Does this work correctly cross-platform? We had to tweak this good amount to correctly get it working.

Reply:
I have not heard of quotes being a problem, cross platform. The reason I touched it at all was to add the double dash, because it was confusing me. I thought that `--fix 'src/**/` was a parameter.

Reply:
And the single quotes are just cleaner. 

But I just changed it, and added double quotes to the new script as well.

Reply:
You beat me to it!

Reply:
Ah fine for now, unless they make troubles. Easy to fix anyway if needed.